packages/server-wallet/src/wallet/store.ts

```diff
@@ -31,6 +34,16 @@ const throwMissingChannel: MissingAppHandler<any> = (channelId: string) => {
 };
 
 export const Store = {
+  getFirstParticipant: async function(): Promise<Participant> {
+    const signingKey = await SigningWallet.query().first();
+    if (!signingKey) throw new StoreError(StoreError.reasons.missingSigningKey);
+    return {
+      participantId: 'unspecified',
```

Comment:
For the time being, we should perhaps continue with making the participant ID equal to the signing address?
```suggestion
      participantId: signingKey.address,
```